docs/querying/sql-functions.md

```diff
@@ -2787,70 +2787,222 @@ Returns the following:
+| -- | -- |
+| `{"type":"PercentClear","percentage":55}` | `{"type":"PercentClear","percentage":55,"example_string":123}` |
+
+</details>
+
+[Learn more](sql-json-functions.md)
 
 ## JSON_OBJECT
 
-Constructs a new `COMPLEX<json>` object. The `KEY` expressions must evaluate to string types. The `VALUE` expressions can be composed of any input type, including other `COMPLEX<json>` values. `JSON_OBJECT` can accept colon-separated key-value pairs. The following syntax is equivalent: `JSON_OBJECT(expr1:expr2[, expr3:expr4, ...])`.
+Constructs a new `COMPLEX<json>` object from one or more expressions. 
+The `KEY` expressions must evaluate to string types.
+The `VALUE` expressions can be composed of any input type, including other `COMPLEX<json>` objects.
+The function can accept colon-separated key-value pairs.
 
-* **Syntax**: `JSON_OBJECT(KEY expr1 VALUE expr2[, KEY expr3 VALUE expr4, ...])`
+* **Syntax:** `JSON_OBJECT(KEY expr1 VALUE expr2[, KEY expr3 VALUE expr4, ...])`  
+  or  
+  `JSON_OBJECT(expr1:expr2[, expr3:expr4, ...])`
 * **Function type:** JSON
 
-[Learn more](sql-json-functions.md)
+<details><summary>Example</summary>
+
+The following example creates a new object `combinedJSON` from `continent` in `geo_ip` and `type` in `event`:
+
+```sql
+SELECT
+  JSON_OBJECT(
+     KEY 'geo_ip' VALUE JSON_QUERY(geo_ip, '$.continent'),
+     KEY 'event' VALUE JSON_QUERY(event, '$.type')
+     )
+  as combinedJSON
+FROM "kttm_nested"
+LIMIT 1
+```
+
+Returns the following:
 
+| `combinedJSON` |
+| -- |
+| `{"geo_ip": {"continent": "South America"},"event": {"type": "PercentClear"}}` |
+
+</details>
+
+[Learn more](sql-json-functions.md)
 
 ## JSON_PATHS
 
-Returns an array of all paths which refer to literal values in `expr` in JSONPath format.
+Returns an array of all paths which refer to literal values in an expression, in JSONPath format.
 
-* **Syntax**: `JSON_PATHS(expr)`
+* **Syntax:** `JSON_PATHS(expr)`  
 * **Function type:** JSON
 
-[Learn more](sql-json-functions.md)
+<details><summary>Example</summary>
+
+The following example returns an array of distinct paths in the `geo_ip` nested column:
+
+```sql
+SELECT
+  ARRAY_CONCAT_AGG(DISTINCT JSON_PATHS(geo_ip)) AS geo_ip_paths
+from "kttm_nested"
+```
+
+Returns the following:
+
+| `geo_ip_paths` |
+| -- |
+| `[$.city, $.continent, $.country, $.region]` |
 
+</details>
+
+[Learn more](sql-json-functions.md)
 
 ## JSON_QUERY
 
-Extracts a `COMPLEX<json>` value from `expr`, at the specified `path`.
+Extracts a `COMPLEX<json>` value from an expression at a specified path.
 
-* **Syntax**: `JSON_QUERY(expr, path)`
+* **Syntax:** `JSON_QUERY(expr, path)`  
 * **Function type:** JSON
 
-[Learn more](sql-json-functions.md)
+<details><summary>Example</summary>
+
+The following example returns the values of `percentage` in the `event` nested column:
+
+```sql
+SELECT
+   "event",
+   JSON_QUERY("event", '$.percentage')
+FROM "kttm_nested"
+LIMIT 2
+```
+
+Returns the following:
```

Comment:
good example! I like how the event column shows how it's being "unnested" in the percent column.